manifests/service.pp

```diff
@@ -2,22 +2,37 @@
 # @api private
 class pulpcore::service {
 
+  ['gunicorn', 'rq'].each |$bin| {
+    file { "/usr/libexec/pulpcore/${bin}":
+      ensure  => file,
+      content => "#!/bin/bash\nexec ${bin} \"$@\"\n",
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0755',
+      # https://github.com/pulp/pulpcore-selinux/pull/13
```

Comment:
This is not clear to me if it means this exists until the pull request is merged or depends upon the pull request. Can you update the wording in the comment to clarify?

Reply:
That PR would add the default, now we need to explicitly set it.

Reply:
Sorry, I'm still not clear the implications on this PR. Does this PR need the pulpcore-selinux PR?

Reply:
No, it does not need that PR. After that PR is released, the line below it can be dropped.